code/style.css

```diff
@@ -1,7 +1,414 @@
+  color: gold;
+  font-size: 35px;
+  margin: 10px;
+}
+
+.info-text {
+  font-size: 20px;
+  margin: 0;
+  font-family: 'Open Sans', sans-serif;
+  text-align: center;
+  color: rgb(255, 255, 255);
+}
+
+.sign-up-text {
+  text-align: center;
+  font-family: 'Open Sans', sans-serif;
+  color: gold;
+  font-size: 27px;
+  margin: 10px;
+}
+
+/* sign up form */
+
+.interaction-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 1050px;
+  background-color: rgb(0, 0, 0);
+  padding: 10px 30px 0px 30px;
+}
+
+.submit-container {
+  border: 3px solid rgb(255, 255, 255);
+  width: 460px;
+  height: 470px;
+  margin: 0px;
+  padding: 20px;
+}
+
+.sign-up-box {
+  width: 300px;
+  background-color: rgb(0, 0, 0);
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+}
+
+.fill-in-form-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  background-color: rgb(0, 0, 0);
+  width: 460px;
+  height: 250px;
+  margin: -1px;
+}
+
+.radiobutton-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 350px;
+  height: 100px;
+  background-color: rgb(0, 0, 0);
+  font-family: 'Open Sans', sans-serif;
+  font-size: 15px;
+  color: white;
+}
+
+h3 {
+  color: gold;
+}
+
+.radiobuttons {
+  display: flex;
+  justify-content: space-between;
+  width: 250px;
+}
+
+.input-text {
+  width: 180px;
+}
+
+label {
+  margin-bottom: 20px;
+}
+
+label > p {
+  font-family: 'Open Sans', sans-serif;
+  color: white;
+  margin: 0;
+}
+
+.checkbox-box {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(0, 0, 0);
+  width: 460px;
+  margin: -1px;
+  color: white;
+}
+
+.checkbox-box-title {
+  display: flex;
+  justify-content: center;
+  font-family: 'Open Sans', sans-serif;
+  color: gold;
+  font-size: 15px;
+  width: 460px;
+}
+
+.checkbox {
+  margin-left: 10px;
+  margin-bottom: 10px;
+}
+
+.checkbox-text {
+  font-family: 'Open Sans', sans-serif;
+  color: white;
+  font-size: 15px;
+  margin-top: 10px;
+}
+
+.button-box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(0, 0, 0);
+  width: 460px;
+  height: 100px;
+  margin: -1px;
+}
+
+button {
+  font-family: 'Open Sans', sans-serif;
+  background-color: gold;
+  font-size: 20px;
+  width: 125px;
+  height: 75px;
+  /*margin-top: 20px;*/
+}
+
+
+/* FAQ SECTION */
+
+.accordion-container {
+  width: 460px;
+  height: 470px;
+  margin: 0;
+  padding: 20px;
+}
+
+.accordion-title {
+  display: flex;
+  justify-content: center;
+  font-family: 'Open Sans', sans-serif;
+  color: gold;
+  font-size: 15px;
+  width: 460px;
+  background-color: rgb(0, 0, 0);
+  margin: 0;
+}
+
+.accordion {
+  border: 3px solid gold;
+  margin-bottom: 10px;
+  padding: 10px;
+
+}
+
+.question-header {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 15px;
+  margin: 0px;
+  color: white;
+}
```

Comment:
If you add `cursor: pointer;` it will let the user know the header is clickable :)